packages/site/src/data/angular/courses.ts

```diff
@@ -53,4 +62,141 @@ export const courses: Course<typeof courseTags[number]>[] = [
 		href: "https://www.rxjs-fruits.com/subscribe",
 		tags: ["rxjs"],
 	},
+	{
+		title: "JavaScript Marathon: Using Angular Libraries in an Nx Monorepo",
+		author: "This Dot Labs",
```

Comment:
For all the authors, let's attribute to the presenter and not This Dot Labs. I think the image is fine as it's presented by our Media account, but I'd prefer to credit the presenter for the content.